index.html

```diff
@@ -0,0 +1,357 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <meta content="detailpagina funda" name="description">
+
+    <!-- Link to CSS -->
+    <link href="styles/style.css" rel="stylesheet" type="text/css">
+
+    <!-- Link to boxicons -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+
+
+    <!-- Link to JavaScript script -->
+
+    <script defer src="scripts/v2.js">
+
+    </script>
+    <!--geen defer vanwege error in w3c validator-->
+
+    <title>Detailpagina</title>
+    <!--    hier werkt de rode knp weer niet oorzaak niet bekend-->
```

Comment:
Not sure what red button you are referring to and why it should be in the `head`. I will ignore this for now, because maybe you can figure out the issue by yourself or when pairing with other students. However, if you're getting blocked on this issue, let us know and we can maybe have a look.

```suggestion
```